games-emulation/picodrive_libretro/picodrive_libretro-1.92_20180419.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="A port of PicoDrive, a Sega 16bit emulator to the libretro API"
+DESCRIPTION="PicoDrive is a Megadrive / Genesis / Sega CD / Mega CD / 32X \
+emulator, focusing on performance."
+HOMEPAGE="http://notaz.gp2x.de/pico.php"
+COPYRIGHT="2005-2018 notaz, the libretro team"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+srcGitRev="eaac037a9e331e8b26a77239d8dd96b76f86515e"
+SOURCE_URI="https://github.com/libretro/picodrive/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="af6706f90bfd1de07e1f3cd7afe2e1e9ac37f0f176df5ccc5d80fdc4a49e4fea"
+SOURCE_FILENAME="libretro-picodrive-${portVersion/_/-}-$srcGitRev.tar.gz"
+SOURCE_DIR="picodrive-$srcGitRev"
+ADDITIONAL_FILES="picodrive_libretro.info.in"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	picodrive_libretro$secondaryArchSuffix = $portVersion
+	addon:picodrive_libretro$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	retroarch$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
```

Comment:
Thanks @kwyxz for this PR!

```diff
-	cmd:git
```
(It's not required and the resulting package is the same _with_ vs _without_ `cmd:git`.)

Everything else is OK :)